docs/diagrams/SaveSequenceDiagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+hide footbox
+actor User
+
+User -> ":Ui" ++: "exit"
+create ":MeetingJio"
+":Ui" -> ":MeetingJio" --: executeCommand("exit", masterTimetable)
+":MeetingJio" -> ":MeetingJio" ++: exit()
+create ":StorageFile"
+":MeetingJio" -> ":StorageFile" ++: saveData(masterTimetable)
```

Comment:
Is the saveData() method instantiating a Storage object here since its pointing to the initial object?  This is also confusing as usually the constructor method instantiates the object. Method calls should be pointing to the activation bar instead!

Reply:
![image](https://user-images.githubusercontent.com/77721568/160958697-febeb0b1-67fb-4a0a-b0ee-b490b016f293.png)